lib/spack/spack/test/environment_modifications.py

```diff
@@ -295,11 +352,20 @@ def test_source_files(files_to_be_sourced):
 
     assert len(modifications["PATH_LIST"]) == 3
     assert isinstance(modifications["PATH_LIST"][0], RemovePath)
-    assert modifications["PATH_LIST"][0].value == "/path/third"
+    if is_windows:
+        assert modifications["PATH_LIST"][0].value == "\\path\\third"
+    else:
+        assert modifications["PATH_LIST"][0].value == "/path/third"
```

Comment:
sections like this are going to get cleaned up by using system specific separators